subsys/bluetooth/controller/ll_sw/ull_llcp.c

```diff
@@ -874,8 +874,11 @@ uint8_t ull_cp_cis_create(struct ll_conn *conn, struct ll_conn_iso_stream *cis)
 	struct ll_conn_iso_group *cig;
 	struct proc_ctx *ctx;
 
-	if (conn->lll.handle != cis->lll.acl_handle) {
-		return BT_HCI_ERR_CMD_DISALLOWED;
+	if (!conn->llcp.fex.valid) {
+		/* No feature exchange was performed so initiate before CIS Create */
+		if (ull_cp_feature_exchange(conn) != BT_HCI_ERR_SUCCESS) {
+			return BT_HCI_ERR_CMD_DISALLOWED;
+		}
```

Comment:
- Do we want to do auto feature exchange?
- How is controller initiated control procedure differentiated from Host initiated?
- Where is the CIS support checked once feature exchange completes?
- How and what event to be returned if remote does not support CIS?

Reply:
It also ties well into

> 4.5.11 Control PDU length management
> The Link Layer shall not transmit a packet containing an LL Control PDU with a
> CtrData field longer than 26 octets until it has successfully completed a Feature
> Exchange procedure (see Section 5.1.4) on the same connection.
>
> If the Link Layer in the Central role supports receiving LL Control PDUs with a
> CtrData field longer than 26 octets, it should initiate the Feature Exchange
> procedure on each connection.
>
> Note: As specified in Section 4.6, once the feature exchange has completed,
> the Link Layer must not use a procedure that the peer did not mark as
> supported. Therefore the Link Layer will never transmit an LL Control PDU with
> a CtrData field longer than 26 octets to a device that does not support it.

And LL_CIS_REQ is 35 bytes.

Reply:
Re. 1: I guess they are not. I think that should not be a problem. As far as I understand - an extra FEAT_EXCH could then be performed, but that ought not be a problem.
Re. 2: It is not, and that is a problem! A solution could be to implement at check for feature support in the CIS Create procedure (before sending CIS_REQ) and complete with UNSUPPORTED in case it is? Then if a CIS create is enqueued 'pending' completion of FEAT_EXCH would be aborted, signaling 
Re. 3: see above?

Reply:
Changes implemented.